xtask/src/lib.rs

```diff
@@ -53,6 +53,7 @@ fn reformat(text: impl std::fmt::Display) -> Result<String> {
     write!(rustfmt.stdin.take().unwrap(), "{}", text)?;
     let output = rustfmt.wait_with_output()?;
     let stdout = String::from_utf8(output.stdout)?;
+    // TODO: update the preable: replace ra_tools with the relevant path
```

Comment:
We have a no todo check, but seem like ir do not parse xtask itself? 😃 

https://github.com/rust-analyzer/rust-analyzer/blob/cdcb3d3833d3d5b37b2cd4dac91a6e9366f20aea/xtask/tests/tidy-tests/cli.rs#L36

Reply:
Seems like it.

@Veetaha We tend to use `// FIXME` for things like this, just for future use.

Reply:
@edwin0cheng @kiljacken but why no to-dos? I can even see them all over the codebase